app/services/GeoServer.js

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
I'm hoping everyone follows a similar pattern here. I've created this central file so that everyone can add their calls to the endpoint in here. Then we can just import this single file and have access to all the endpoints.